import_export/resources.py

```diff
@@ -699,9 +699,21 @@ def before_import_row(self, row, row_number=None, **kwargs):
         """
         pass
 
-    def after_import_row(self, row, row_result, row_number=None, **kwargs):
+    def after_import_row(
+        self, row, row_result, row_number=None, original=None, **kwargs
```

Comment:
I might be misremembering but I thought this provided the original and the new when I looked at this last.  What is the thought on if someone wants to know what the new value is? I recalled that your example use case was the new book status of out of print. Guessing it's something obvious I'm missing though :).

Reply:
Hi Ryan

Thanks for reviewing.  In short, I changed to override `after_import_row()` not `after_import_instance()` because it is cleaner in my view.

I was originally overriding `after_import_instance()` (note this method is misnamed, I raised a separate issue for that).  The problem with overriding `after_import_instance()` is that you can check the changed values by looking at the row, but if you want extra workflow (which is my use case), then this has to execute before the base instance is persisted (or you have to add some hacky attributes or similar).

I looked at it again and thought that overriding `after_import_row()` is cleaner, because the `original` object would be available at this point, and makes it very easy for comparisons.  Also `after_import_row()` is called after the base instance is persisted.

I updated the example in the docs - you can compare `original` with `instance` - the `row_result` contains the instance.  I'll update the method docstring to make this clearer.

Reply:
Am thinking it might be more consistent to set `original` onto `RowResult`, instead of passing it in as a param.. I think I will probably make that change

Reply:
That makes great sense. Sorry I didn't grasp the details of row result :) I'm on board with that change and fine to merge if everything passes tests. Thanks again for this cool work! Pipeline step hooks are awesome flexibility.

Reply:
On a separate note - please can you email me on mrhegarty@gmail.com - just need to run something by you